tests/AutoReview/ProjectCodeTest.php

```diff
@@ -65,6 +65,7 @@
         \PhpCsFixer\Runner\FileFilterIterator::class,
         \PhpCsFixer\Runner\FileLintingIterator::class,
         \PhpCsFixer\StdinFileInfo::class,
+        \PhpCsFixer\Test\Assert\AssertTokensTrait::class,
```

Comment:
underlaying reason of creating `trait` is to provide reusable feature that is not in context of abstract entity. thus, that feature could be tested.
after said that, we introduced this trait to test other bugfix, let us not focus in this PR on assertion2trait extraction as we need to update assertion as well (to handle cleared tokens)